lib/Module/Build/Tiny.pm

```diff
@@ -209,8 +209,8 @@ than 120, yet supports the features needed by most distributions.
 =head2 Directory structure
 
 Your .pm, .xs and .pod files must be in F<lib/>.  Any executables must be in
-F<script/>.  Test files must be in F<t/>. Dist sharedirs must be in F<share/>.
-
+F<script/> or F<bin/>.  Test files must be in F<t/>. Dist sharedirs must be in
```

Comment:
Hmm, I thought we were moving towards preferring `script`? Is there a compelling reason to add `bin` now?

Reply:
People have asked for it before, but I now took it out because it may lead to confusion wrt static install.